### ISSUE TITLE
SDL2 2.30.0 has issues with Controller buttons not registering 

### ISSUE THREAD

**Issue by Zombie-Ryushu** [libsdl-org/SDL#9019]
SDL2 has issues with Controller buttons not registering.

A number of games, and emulators are broken reading button inputs. Game pads are Detected, but certain buttons don't work. It upgrades SDL2 2.30.0. USB Controllers stop functioning correctly.

This affects:

Wine
Cannnonball
PPSSPP
PCSX2.

If I use LD_LIBRARY_PATH to load SDL 2.0.8, all the buttons on my gamepad work as intended again.

**Comment by slouken** [comment]
Which controllers are affected, and which buttons?

Can you give step by step complete instructions on how to reproduce this? Or even better, can you debug what's going on there yourself?

**Comment by Zombie-Ryushu** [comment]
Yes, take an application such as ppsspp or Wine, built with SDL 2.0.8 in mind. Try and map buttons in Wine games or any game effected this. Examples are:

Wine
PPSSPP
Cannonball.

install libsdl 2.33.0 in /usr/lib64 and 2.0.8 in /usr/local/lib64. Direct a Symlink to libSDL-2.0.so.0
ine, run Wine control.  test the buttons.

I use a: Bus 001 Device 112: ID 0738:8828 Mad Catz, Inc. PS3 RF pad

Button 3, 4, 12 and 13 will not register.
